.gitlab-ci.yml

```diff
@@ -48,6 +48,8 @@ check-clippy:
   script:
     - rustup component add clippy --toolchain stable-x86_64-unknown-linux-gnu
     - time cargo +stable clippy --locked -- -Dwarnings
+    - time cargo +stable clippy --locked -p parity-scale-codec-derive -- -Dwarnings
```

Comment:
Can you not use something like `--workspace --tests`?

Reply:
That would run clippy on all of the integration tests, which at this point are not clippy clean, and at this point I didn't feel like also cleaning up all of the tests.